docs/examples/Validation Examples.ipynb

```diff
@@ -127,23 +127,10 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 3,
+   "execution_count": null,
```

Comment:
unfortunately, the validation examples need to be run entirely manually (or run with `--continue-on-error` or something similar).

Reply:
I didn't notice this, good catch, I will fix it.